sdk/core/abort-controller/package.json

```diff
@@ -7,15 +7,19 @@
   "module": "dist-esm/src/index.js",
   "scripts": {
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
-    "build:es6": "tsc -p tsconfig.json",
-    "build:nodebrowser": "rollup -c 2>&1",
     "build:types": "downlevel-dts types/src types/3.1",
-    "build:test": "rollup -c rollup.test.config.js 2>&1",
-    "build": "npm run build:es6 && npm run build:nodebrowser && npm run build:types",
+    "build:ts": "tsc -p .",
```

Comment:
If I just look at the `"unit-test:node"` command in different libraries, there are multiple variations of it.
Did you intend to tackle those inconsistencies in this PR?

Reply:
Abort-controller is different, it has additional env variables. Rest all packages follow template package.json.